src/HydrationRepo.js

```diff
@@ -0,0 +1,27 @@
+class HydrationRepo {
+  constructor(hydrations) {
+    this.allHydrations = hydrations;
+  }
+  userLifetime(id) {
+    let userHydrations = this.allHydrations.filter(hydration => {
+      return hydration.userID === id;
+    })
+    let sumHydrations = userHydrations.reduce((total, day) => {
+      return total += day.numOunces;
+    }, 0)
+    return Math.floor(sumHydrations / userHydrations.length);
+  }
```

Comment:
Minor change but may want to make this method an action statement... could be `calculateUserLifetimeHydration` - I would see the current name being more representative of a variable rather than a method.